cartridge-cli.lua

```diff
@@ -1651,7 +1676,240 @@ local function run_hook(dir, filename)
     return true
 end
 
-local function build_application(dir)
+-- * --------------------------- Dockerfile parts --------------------------
+
+local function tarantool_repo_version()
+    local parts = _TARANTOOL:split('.')
+    return string.format('%s_%s', parts[1], parts[2])
+end
+
+local function construct_install_tarantool_dockerfile_part()
+    local install_tarantool_dockerfile_part
+    if app_state.tarantool_is_enterprise then
+        local version_filepath = fio.pathjoin(get_tarantool_dir(), 'VERSION')
+        local version_filepath_content = read_file(version_filepath)
+
+        local sdk_version = string.match(version_filepath_content, 'TARANTOOL_SDK=(%S+)\n')
+        if sdk_version == nil then
+            return nil, string.format('Failed to get SDK version from %s file', version_filepath)
+        end
+        sdk_version = sdk_version:gsub('-macos', '')
```

Comment:
what? why? I think macos shouldn't be mention at all

Reply:
When you develop locally on your macos, your TE VERSION file contains string like `TARANTOOL_SDK=2.2.1-65-g34831ed-macos`. This version is used to detect TE version that should be delivered in the result package, and it shouldn't contain `macos` suffix. In a future, we plan to allow user to specify Tarantool version for the result package, but now it's detected from the version used for development.